src/Microsoft.Framework.Runtime.Internals/Microsoft.Framework.Runtime.Internals.xproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="__ToolsVersion__" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Ick. Run one of the scripts to fix this file up e.g. \\GarnetA\share\UpdateXProjectFiles\UpdateXProjectFiles.ps1